EndlessClient/Rendering/ContextMenuRenderer.cs

```diff
@@ -122,6 +126,7 @@ private enum MenuAction
         public override void Initialize()
         {
             base.Initialize();
+            _sfxPlayer.PlaySfx(SoundEffectID.ButtonClick);
```

Comment:
I'd avoid a `_sfxPlayer` call here, the paradigm for `Initialize()` is really more for setting up the control. Maybe this belongs in the `ContextMenuRendererFactory`?

Reply:
Or even better, where the factory method gets called. Something about having an SFX plays with every call to `Initialize` on a class feels wrong to me, I'd rather have it tied to an action instead

Reply:
Resolved in split-out PR for SFX stuff